sp800-63a/sec5_proofing.md

```diff
@@ -83,28 +83,28 @@ The goal of identity verification is to confirm and establish a linkage between
+|:---:|:------------------------------|
 |Unacceptable|Evidence verification was not performed or verification of the evidence failed. Unable to confirm that the applicant is the owner of the claimed identity.|
 |Weak|The applicant has been confirmed as having access to the evidence provided to support the claimed identity.|
-|Fair|- The applicant's ownership of the claimed identity has been confirmed by:<br>&nbsp;&nbsp;&nbsp;&nbsp;- KBV.  See [Section 5.3.2](#kbv) for more details. <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**<br>&nbsp;&nbsp;&nbsp;&nbsp;- a physical comparison of the applicant to the identity evidence. Physical comparison performed remotely SHALL include presentation attack detection as specified in [[SP 800-63B, Section 5.2.3]](sp800-63b.html/#biometric_use). <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**<br>&nbsp;&nbsp;&nbsp;&nbsp;- biometric comparison of the applicant to the identity evidence. Biometric comparison performed remotely SHALL adhere to the appropriate requirements as specified in [[SP 800-63B, Section 5.2.3]](sp800-63b.html/#biometric_use). |
-|Strong|- The applicant's ownership of the claimed identity has been confirmed by: <br>&nbsp;&nbsp;&nbsp;&nbsp;- physical comparison, using appropriate equipment, to a photograph or image, on all applicable evidence. Physical comparison performed remotely SHALL include presentation attack detection as specified in [[SP 800-63B, Section 5.2.3]](sp800-63b.html/#biometric_use). <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**<br>&nbsp;&nbsp;&nbsp;&nbsp;- biometric comparison, using appropriate equipment, of the applicant to the strongest piece of identity evidence. Biometric comparison performed remotely SHALL adhere to the appropriate requirements as specified in [[SP 800-63B, Section 5.2.3]](sp800-63b.html/#biometric_use).|
-|Superior|- The applicant's ownership of the claimed identity has been confirmed by biometric comparison of the applicant to the strongest pieces of identity evidence, using appropriate equipment. Biometric comparison performed remotely SHALL adhere to the appropriate requirements as specified in [[SP 800-63B, Section 5.2.3]](sp800-63b.html/#biometric_use).|
+|Fair|- The applicant's ownership of the claimed identity has been confirmed by:<br>&nbsp;&nbsp;&nbsp;&nbsp;- KBV.  See [Section 5.3.2](#kbv) for more details. <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**<br>&nbsp;&nbsp;&nbsp;&nbsp;- a physical comparison of the applicant to the identity evidence. Physical comparison performed remotely SHALL adhere to all requirements as specified in [[SP 800-63B, Section 5.2.3]](sp800-63b.html/#biometric_use). <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**<br>&nbsp;&nbsp;&nbsp;&nbsp;- biometric comparison of the applicant to the identity evidence. Biometric comparison performed remotely SHALL adhere to all requirements as specified in [[SP 800-63B, Section 5.2.3]](sp800-63b.html/#biometric_use). |
```

Comment:
Nit: "adhere to all requirements as specified" -> "adhere to all requirements specified"

Same correction several places in the table.